portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,106 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#background-block {
+    background-color: #e897c3 ;
+}
+
+.logo {
+  position: absolute;
+  left: 15px;
+  top: 105px;
+}
+
+.headshot {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+.h1 { 
+    font-family: "Rockwell Extra Bold", "Rockwell Bold", monospace; 
+    font-size: 40px; 
+    font-style: normal; 
+    font-variant: normal; 
+    font-weight: 700; 
+    line-height: 42.4px; 
+    white-space: nowrap;
+    margin-top: 115px;
+}
+
+.h3 { 
+    font-family: Rockwell, "Courier Bold", Courier, Georgia, Times, "Times New Roman", serif;
+    font-size: 20px; 
+    font-style: normal; 
+    font-variant: normal; 
+    font-weight: 100; 
+    line-height: 21.4px; 
+    white-space: nowrap;
+    margin-top: 30px;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  text-align: center;
+}
+
+
+/* Style the header with a grey background and some padding */
+.header {
+  overflow: hidden;
+  background-color: #f1f1f1;
+  padding: 20px 10px;
+}
+
+/* Style the header links */
+.header a {
+  float: left;
+  color: black;
+  text-align: center;
+  padding: 12px;
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 25px;
+  border-radius: 4px;
+}
+
+/* Style the logo link (notice that we set the same value of line-height and font-size to prevent the header to increase when the font gets bigger */
+.header a.logo {
```

Comment:
From https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors:

Avoid qualifying ID and class names with type selectors.

Unless necessary (for example with helper classes), do not use element names in conjunction with IDs or classes.